src/main/plugins/org.dita.html5/sass/_lists.scss

```
@@ -22,14 +22,10 @@ ul.simple {
   margin-top: 1em;
 }
 
-*[compact="yes"] > li {
+.compact > li {
```

For backwards compatibility, let's preserve the attribute selector to ensure the rule continues to work on any files generated by previous toolkit versions:


```suggestion
*[compact="yes"] > li,
.compact > li {
```